src/routes/controllers/productsControllers/createProduct.js

```diff
@@ -3,67 +3,68 @@ import validateData from '../../../helpers/validateData';
 import ProductModel from '../../../models/product';
 
 const imageValidation = Joi.object({
-  name: Joi.string(),
-  src: Joi.string(),
+	altName: Joi.string(),
+	name: Joi.string(),
+	src: Joi.string(),
 });
 
 const specsTableValidation = Joi.object({
-  heading: Joi.string(),
-  rows: Joi.array().items(Joi.array().items(Joi.string())),
+	heading: Joi.string(),
+	rows: Joi.array().items(Joi.array().items(Joi.string())),
 });
 
 const validationSchema = Joi.object({
-  title: Joi.string().required(),
-  vendor: Joi.string(),
-  description: Joi.string(),
-  images: Joi.array().items(imageValidation),
-  quantity: Joi.number(),
-  isInStock: Joi.boolean(),
-  categories: Joi.array().items(Joi.string()),
-  subCategories: Joi.array().items(Joi.string()),
-  price: Joi.number(),
-  sale: Joi.number(),
-  specifications: Joi.array().items(specsTableValidation),
+	title: Joi.string().required(),
```

Comment:
This is technically the same as before. I dont know why git would think these codes as changes

Reply:
they are same changes but looks different because of prettier config